x-pack/metricbeat/docker-compose.yml

```diff
@@ -23,11 +23,11 @@ services:
   kibana:
     # Copied configuration from OSS metricbeat because services with depends_on
     # cannot be extended with extends
-    image: docker.elastic.co/integrations-ci/beats-kibana:${KIBANA_VERSION:-7.4.0}-1
+    image: docker.elastic.co/integrations-ci/beats-kibana:${KIBANA_VERSION:-7.5.2}-1
```

Comment:
If we upgrade kibana here we should also do it in the OSS `docker-compose.yml`, and we should also upgrade ES.

The issue with the lambda dashboard is already fixed in master (https://github.com/elastic/beats/pull/15682) so it should be fixed in the next update we do.

Reply:
Will remove.